File: 20/css.rb
#encoding: utf-8
require 'selenium-webdriver'

dr = Selenium::WebDriver.for :chrome
file_path = 'file:///' + File.expand_path(File.join('.', 'css.html'))
dr.get file_path

link = dr.find_element(id: 'tooltip')
puts link.css_value(:color)

puts dr.find_element(:tag_name, 'h3').css_value('font')

dr.quit()

File: 20/Css.java
import java.io.File;
import java.util.List;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;


public class Css {

	public static void main(String[] args) throws InterruptedException {
		WebDriver dr = new ChromeDriver();
		
		File file = new File("src/css.html");
		String filePath = "file:///" + file.getAbsolutePath();
		System.out.printf("now accesss %s \n", filePath);
		
		dr.get(filePath);
		Thread.sleep(1000);
		
		WebElement link = dr.findElement(By.id("tooltip"));
		
		System.out.println(link.getCssValue("color"));
		
		System.out.println(dr.findElement(By.tagName("h3")).getCssValue("font"));
		
		Thread.sleep(1000);
		System.out.println("browser will be close");
		dr.quit();	
	}

}
